docs/ocis/release_notes.md

```diff
@@ -7,6 +7,26 @@ geekdocEditPath: edit/master/docs/ocis
 geekdocFilePath: release_notes.md
 ---
 
+## ownCloud Infinite Scale 1.6.0 Technology Preview
+
+To get the full potential out of the microservice architecture, version 1.6.0 introduces a dynamic service registry to ownCloud Infinite Scale. The dynamic service registry facilitates the configuration and contributes to the scalability of the platform. ownCloud Web 3.1.0 has received further improvements for accessibility like keyboard navigation and it comes with performance improvements by loading certain elements asynchronously.
```

Comment:
The new dynamic registry is new to REVA, in oCIS we already used it before, right @refs? I think it could be mentioned somehow that we helped to make REVA more cloud native ;-)